core/kotlinx-coroutines-core/src/Executors.kt

```diff
@@ -98,7 +92,7 @@ private class ResumeUndispatchedRunnable(
  * An implementation of [DisposableHandle] that cancels the specified future on dispose.
  * @suppress **This is unstable API and it is subject to change.**
  */
-public class DisposableFutureHandle(private val future: Future<*>) : DisposableHandle {
+class DisposableFutureHandle(private val future: Future<*>) : DisposableHandle {
```

Comment:
@elizarov any reason it's not internal?

Reply:
Frankly I don't remember. If it compiles with `internal`, then it should be `internal`.